integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/mortgage/MortgageSpark.scala

```diff
@@ -426,12 +426,23 @@ object Main {
     val acqPath = args(2)
     val output = args(3)
 
+    // extend args to support csv/orc/parquet dataset
+    val format = if (args.length > 4) args(4) else "parquet"
```

Comment:
Nit: We really should throw an error if there's more than 5 arguments, because at that point we're silently dropping arguments the user is expecting to be honored.

Reply:
Agree, updated